src/model/databse.config.ts

```diff
@@ -0,0 +1,15 @@
+import { TypeOrmModuleOptions } from '@nestjs/typeorm';
+// package.json has to install mysql2 and mysql must be deleted.
+const databaseConfig: TypeOrmModuleOptions = {
+  type: 'mysql',
+  host: '127.0.0.1',
+  port: 3306,
+  username: 'root',
+  password: '',
+  database: 'friends_nest',
+  autoLoadEntities: true,
+  entities: [__dirname + '/**/*.entity{.ts,.js}'],
+  synchronize: true,
```

Comment:
Need to test this and use it carefully, from the official doc:
```
 Indicates if database schema should be auto created on every application launch. Be careful with this option and don't use this in production - otherwise you can lose production data.
```

Reference -- https://typeorm.io/data-source-options#common-data-source-options

Reply:
done